standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -457,6 +457,10 @@ public boolean isCompatibleWith(Configuration conf) {
     }
     boolean compatible = true;
     for (ConfVars oneVar : MetastoreConf.metaVars) {
+      // METASTORE_TOKEN_SIGNATURE is no need to check
```

Comment:
We can just remove `TOKEN_SIGNATURE` from `MetastoreConf.metaVars` list:
https://github.com/apache/hive/blob/1c5d9b90d960af8b84a1278e2faf8073a352c191/standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java#L189

Reply:
do you mean I can save the token with a global variable ? because it is used when open connection
![image](https://user-images.githubusercontent.com/20615138/225179850-de90b2a9-0f39-4499-9eae-4801b9faf627.png)

Reply:
No, I mean we can remove the `ConfVars.TOKEN_SIGNATURE` from following array:
https://github.com/apache/hive/blob/1c5d9b90d960af8b84a1278e2faf8073a352c191/standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java#L157-L165

Reply:
thanks for explain, I get it,  and update the commit code